server/svix-server/src/v1/endpoints/application.rs

```diff
@@ -255,12 +259,16 @@ async fn create_application(
     Ok((StatusCode::CREATED, Json((app, metadata).into())))
 }
 
+const GET_APPLICATION_DESCRIPTION: &str = "Get an application.";
+
 async fn get_application(
     permissions::ApplicationWithMetadata { app, metadata }: permissions::ApplicationWithMetadata,
 ) -> Result<Json<ApplicationOut>> {
     Ok(Json((app, metadata).into()))
 }
 
+const UPDATE_APPLICATION_DESCRIPTION: &str = "Update an application.";
```

Comment:
Adding my 2c since I approved the PR, but I really liked the descriptions being placed right next to the function. It's much easier to follow when the documentation is colocated with the implementation.

Reply:
>Also, can we not derive it from the function name? That's what we do in Python I think.
For this, I assume you mean the ID and not the description (which in Python is the function's docstring). Though either for the function's name _or_ its doc comment, I can't think of a way of doing it without macros.

Reply:
> Adding my 2c since I approved the PR, but I really liked the descriptions being placed right next to the function. It's much easier to follow when the documentation is colocated with the implementation.

Yeah, it's nicer when it's side to side if everything was there, but see what I wrote. I think what will end up happening is that part of it is going to be side-by-side and some not, and that will suck. I guess we could make a structure that we just extend and then keep it side-by-side though. OK, let's see how this grows no need to prematurely optimize.

Reply:
> > Also, can we not derive it from the function name? That's what we do in Python I think.
> 
> For this, I assume you mean the ID and not the description (which in Python is the function's docstring). Though either for the function's name _or_ its doc comment, I can't think of a way of doing it without macros.

I was referring to the summary (which is what you did here, no?). Both the ID and summary (or maybe it's called title?) are automatically generated in Python.

Reply:
>I was referring to the summary (which is what you did here, no?). Both the ID and summary (or maybe it's called title?) are automatically generated in Python.

No, summary and description are separate. There are 3 things at play:
* Operation ID - This is a machine readable identifier, and this is what's used in code generation for the libs. I _think_ this is generated based on the Python identifier but I'm not sure tbh.
* Summary - The short (few-words) text of the operation, e.g. "List Applications", "Rotate Endpoint Secrets", etc. This looks like it's set either based on the function name in python, but it's also manually overridden in some cases.
* Description - The lengthier explanation, set by the docstring in python. That is what this PR is doing.

Reply:
Gotcha. OK, I probably just looked at a few short descriptions and thought this was "summary". 

Cool, so ignore me!